dba_mututation.py

```diff
@@ -0,0 +1,367 @@
+        #print(atom.GetAtomicNum(), atom.GetIsotope(),atom.GetIdx())
+        if atom.GetAtomicNum() == 0 and atom.GetIsotope() == 98:
+            #print('frag3 is PBAglc4mut')
+            PBAglc4mut = frag3 
+            break
+        elif atom.GetAtomicNum() == 0 and atom.GetIsotope() == 99:
+            #print('frag3 is PBAfrc4mut')
+            PBAfrc4mut = frag3
+            break
+        elif atom.GetAtomicNum() == 0 and atom.GetIsotope() == 96:
+            for atom2 in frag3.GetAtoms():
+                if atom2.GetAtomicNum() == 0 and atom2.GetIsotope() == 97:
+                    #print('frag3 is linker4mut')
+                    linker4mut = frag3
+            
+    #print('Fragments classification:',Chem.MolToSmiles(PBAglc4mut),Chem.MolToSmiles(PBAfrc4mut),Chem.MolToSmiles(linker4mut))  
+            
+    p = [4/6, 1/6, 1/6] 
+    mut_operation = np.random.choice(['linker', 'PBAglcyl', 'PBAfrcyl'], p=p)
+    if mut_operation == 'linker':
+        #print("Do mutation on the linker")
+        smiles = Chem.MolToSmiles(linker4mut)
+        linker_mut = mu.mutate(linker4mut,1.0,'middle')
+        linker_mut_smi = Chem.MolToSmiles(linker_mut, kekuleSmiles=True)
+        if linker.HasProp('pKaglcyl') ==1 and linker.HasProp('pKafrcyl') == 1:
+            PBAglc_child, pKaglcyl_child = PBAglc4mut, linker.GetProp('pKaglcyl')
+            PBAfrc_child, pKafrcyl_child = PBAfrc4mut, linker.GetProp('pKafrcyl')
+        else:
+            PBAglc_child = PBAglc4mut
+            PBAglc4mut_smi = Chem.MolToSmiles(PBAglc4mut, kekuleSmiles=True)
+            pKaglcyl_child = pKaFromSimmilarity(PBAglc4mut_smi)
+            PBAfrc_child = PBAfrc4mut
+            PBAfrc4mut_smi = Chem.MolToSmiles(PBAfrc4mut, kekuleSmiles=True)
+            pKafrcyl_child = pKaFromSimmilarity(PBAfrc4mut_smi)
+    elif mut_operation == 'PBAglcyl':
+        #print("Do mutation on the PBAglcyl side")
+        linker_mut = linker4mut
+        linker_mut_smi = Chem.MolToSmiles(linker_mut, kekuleSmiles=True)
+        PBAglc_child, pKaglcyl_child = PBA_mu.Glcyl()
+        if linker.HasProp('pKafrcyl') == 1:
+            PBAfrc_child, pKafrcyl_child = PBAfrc4mut, linker.GetProp('pKafrcyl')
+        else:
+            PBAfrc_child = PBAfrc4mut
+            PBAfrc4mut_smi = Chem.MolToSmiles(PBAfrc4mut, kekuleSmiles=True)
+            pKafrcyl_child = pKaFromSimmilarity(PBAfrc4mut_smi)
+    elif mut_operation == 'PBAfrcyl':
+        #print("Do mutation on the PBAfrcyl side")
+        linker_mut = linker4mut
+        linker_mut_smi = Chem.MolToSmiles(linker_mut, kekuleSmiles=True)
+        if linker.HasProp('pKaglcyl') ==1:
+            PBAglc_child, pKaglcyl_child = PBAglc4mut, linker.GetProp('pKaglcyl')
+        else:
+            PBAglc_child = PBAglc4mut
+            PBAglc4mut_smi = Chem.MolToSmiles(PBAglc4mut, kekuleSmiles=True)
+            pKaglcyl_child = pKaFromSimmilarity(PBAglc4mut_smi)
+        PBAfrc_child, pKafrcyl_child = PBA_mu.Frcyl() 
+
```

Comment:
I added this try function because new molecules are appearing with structural featares not coded in the tweezer_classification function. I think most of this non codded structures are being originated from the crossover operation.